android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -681,16 +681,16 @@ class GeckoEngineSession(
 
             val productRecommendations = response.map { it: Recommendation ->
                 GeckoProductRecommendation(
-                    it.url,
-                    it.analysisUrl,
-                    it.adjustedRating,
-                    it.sponsored,
-                    it.imageUrl,
-                    it.aid,
-                    it.name,
-                    it.grade,
-                    it.price,
-                    it.currency,
+                    it.url!!,
+                    it.analysisUrl!!,
+                    it.adjustedRating!!,
+                    it.sponsored!!,
+                    it.imageUrl!!,
+                    it.aid!!,
+                    it.name!!,
+                    it.grade!!,
+                    it.price!!,
+                    it.currency!!,
```

Comment:
Will remove the double bangs once GV patch lands, as part of this [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1857215)